madnight/githut
# Plot any language

Allows the user to plot any language, not just the top ten.

![image](https://user-images.githubusercontent.com/4427/63231774-719fbe80-c1d5-11e9-8775-b7499ccd233a.png)

Caveat: I'm a MobX newbie. Advice welcome!

Test plan:

* Click the checkboxes in the first column of the table, then see the chart update.
* Click the pull request button and note the same languages stay selected.
